.github/workflows/ci.yml

```diff
@@ -26,12 +26,11 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        #node: ['8', '10', '12', '14', '16', '17']
         #os: [ubuntu-latest, windows-latest, macos-latest]
         # let's make it a little bit simple for now
         # current minimal version will be 12.
         # TODO: investigate the multipe version matrix with single Vertica instance
-        node: ['12', '13', '14', '15', '16', '17']
+        node: ['12', '14', '16', '18', '20']
```

Comment:
Thanks for making the change to test against the stable versions of node. I was not aware of what the different versions originally meant.